app/lib/ui/views/signin/sign_in_screen.dart

```diff
@@ -42,6 +68,8 @@ class SignInScreen extends StatelessWidget {
                     ),
                   ),
                   TextFormField(
+                    key: ObjectKey("Email"),
```

Comment:
When we do system tests, we'll need to pay attention to whether sign in screen and sign up screen exist in the widget tree at the same time. Since they have the same object keys, grabbing an element by the object key `Email` would grab both or we'll have an error at runtime that two widgets exist with the same key. I think keys are global.